<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpportunityFilterWidget</class>
 <widget class="QWidget" name="OpportunityFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Show:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbOpen">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbClosedWon">
       <property name="text">
        <string>Closed Won</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbClosedLost">
       <property name="text">
        <string>Closed Lost</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Priority</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="NullableDateComboBox" name="modifiedAfter">
     <property name="calendarPopup" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="rbAll">
     <property name="text">
      <string>A&amp;ll opportunities</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QComboBox" name="cbMaxNextStepDate">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>One month ago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Today</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End of this week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End of this month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>One month from now</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End of this year</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="Line" name="line_2">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QComboBox" name="cbPriority">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not set</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="rbAssignedTo">
     <property name="text">
      <string>Assigned to</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="NullableDateComboBox" name="modifiedBefore">
     <property name="calendarPopup" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Next step date before</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbCountry"/>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Last modified after</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbAssignee"/>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="rbCountry">
     <property name="text">
      <string>I&amp;n country</string>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="7" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>and before</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NullableDateComboBox</class>
   <extends>QComboBox</extends>
   <header>nullabledatecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
